tests/test_omxplayer.py

```diff
@@ -116,6 +117,11 @@ def test_checks_media_file_exists_before_launching_player(self, *args):
             self.patch_and_run_omxplayer()
             ospath.isfile.assert_called_once_with(self.TEST_FILE_NAME)
 
+    def test_not_checks_url_before_launching_player(self, *args):
```

Comment:
I think a better name is `test_player_doesnt_check_source_path_exists_for_a_url`